data/sql/updates/pending_db_world/rev_1605359195128742200.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1605359195128742200');
+
+UPDATE `spell_proc_event` SET `procFlags` = 0 AND `Cooldown` = 50000 WHERE `entry` = 67653;
```

Comment:
Replace AND with comma

Reply:
Besides that it works good 👍 

Before:
![image](https://user-images.githubusercontent.com/61268368/99866974-1bfe5280-2b8c-11eb-8481-a2f99aac1a94.png)

After:
![image](https://user-images.githubusercontent.com/61268368/99866979-228cca00-2b8c-11eb-8e02-0fc91e28d0f1.png)

Reply:
Why the fuck did i use and xD